src/blocks/extensions/masonry-gallery/index.js

```diff
@@ -35,30 +36,29 @@ const MasonryGallery = createHigherOrderComponent( ( BlockEdit ) => {
 			let document_element = document;
 			// Find the iframe element with the name "editor-canvas" and assign it to getEditorIframe.
 			const getEditorIframe = document.querySelectorAll( 'iframe[name="editor-canvas"]' );
-			const styleElement = document.getElementById( 'uagb-editor-styles' );
 
-			if( getEditorIframe?.length ){
-				// Get the document of the iframe.
-				const iframeDocument = getEditorIframe?.[0]?.contentWindow?.document || getEditorIframe?.[0]?.contentDocument;
-
-				// If the iframe document is present...
-				if ( iframeDocument ) {
-
-					// Set the root element to the iframe document.
-					document_element = iframeDocument;
-
-					// Get the style element with the ID 'uagb-editor-styles' from the iframe document.
-					const _element = document_element.getElementById( 'uagb-editor-styles' );
-
-					// If the HTML style element is not found within the `_document` iframe...
-					if ( !_element ) {
-						if ( styleElement ) {
-
-							// Clone the `styleElement` and append it to the head of the `_document` of iframe.
-							const clonedElement = styleElement.cloneNode( true );
-							document_element.head?.appendChild( clonedElement );
-
-						}
+			if ( ! getEditorIframe?.[ 0 ] ) {
+				return document;
+			}
+			// Get the document of the iframe.
+			const iframeDocument =
+				getEditorIframe[ 0 ]?.contentWindow?.document || getEditorIframe[ 0 ]?.contentDocument;
+
+			// If the iframe document is present...
+			if ( iframeDocument ) {
```

Comment:
Instead of putting this block of code into an if statement, should we early return on the inverse?

```suggestion
			if ( ! iframeDocument ) {
				return document;
			}
```

Reply:
Yes we can have early return in this way, made changes.